pom.xml

```diff
@@ -206,6 +213,32 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>snapshotbuild</id>
+            <activation>
+                <property>
+                    <name>!release</name>
+                </property>
+            </activation>
+            <properties>
+                <build.number>- local build -</build.number>
+                <online.repo>https://openhab.jfrog.io/openhab/online-repo-snapshot/${online.repo.version}</online.repo>
+                <pax.url.suffix>@snapshots</pax.url.suffix>
+            </properties>
+        </profile>
+        <profile>
+            <id>releasebuild</id>
+            <activation>
+                <property>
+                    <name>release</name>
+                </property>
+            </activation>
+            <properties>
+                <build.number>- release build -</build.number>
```

Comment:
@ThomDietrich What was our final decision on this String here? We wanted to use "Release Build" in capitals and without dashes, correct?
@pfink FYI, the current naming with dashes caused issues on openHABian, so @ThomDietrich asked me to change this in future - so this is now the right place to define it :-)

Reply:
Release Build, correct.
@pfink the reason is simple, openHABian will show the version details on ssh login and the dashes were "ugly" while not having any purpose. Currently there is a simple search and replace logic in place:

![grafik](https://user-images.githubusercontent.com/2870104/30854341-a866b0b2-a2b1-11e7-9852-c98f571e9359.png)

Reply:
Your wish is my command. Done ;-)